src/js/components/keyword/table/ResultsTableSection.jsx

```diff
@@ -65,7 +66,7 @@ export default class ResultsTableSection extends React.Component {
                 visibleWidth={this.state.tableWidth} />
         );
 
-        if (!this.props.keywordApplied) {
+        if (this.props.keyword === '') {
```

Comment:
Since `''` is a [falsy value](http://www.ecma-international.org/ecma-262/6.0/#table-10) and the `if` will coerce it to a boolean, you can actually just do `if (this.props.keyword)`.